azure-configurationservice/azure/configuration/aio/azure_configuration_client_async.py

```diff
@@ -0,0 +1,262 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+import os
+import sys
+import platform
+from datetime import datetime
+from msrest.paging import Paged
```

Comment:
Why do we still reference msrest?

Reply:
The Paged is of msrest. No such thing in azure core

Reply:
It is/will be in azure-core.

Reply:
Will create a backlog to update to azure core when azure core has it.